src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,9 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc3-24126-00"
```

Comment:
Does it make sense to put this dependency under the `netstandard1.5` TFM to align with the way Microsoft.NETCore.App is added for a console app?